<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 250, 251)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>601</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-40</y>
      <width>141</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 250, 251);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="today_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>89</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 250, 251);
border:none;
padding:5px;</string>
     </property>
     <property name="text">
      <string>Сегодня </string>
     </property>
    </widget>
    <widget class="QPushButton" name="logbook_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>151</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 250, 251);
border:none;
padding:5px;</string>
     </property>
     <property name="text">
      <string>Журнал</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inbox_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>58</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 250, 251);
border:none;
padding:5px;
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>Входящие</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/inbox.png</normaloff>icons/inbox.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="plans_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(249, 250, 251);
border:none;
padding:5px;</string>
     </property>
     <property name="text">
      <string>Планы</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
